cmake/scripts/windowsstore/ArchSetup.cmake

```diff
@@ -1,5 +1,5 @@
 # Minimum SDK version we support
-set(VS_MINIMUM_SDK_VERSION 10.0.17763.0)
+set(VS_MINIMUM_SDK_VERSION 10.0.18362.0)
```

Comment:
Minimum sdk version controls which version of win10 the app will run on. We shouldn't change this unless we require a feature that's in a newer version of win10.

Reply:
You are right: changing back to 10.0.17763.0  and still compiles OK with 10.0.18362.0

Reverted